### PR TITLE
addoninfo for new add-on suggestion finder "process"

### PR DESCRIPTION
**Pull request by holgerfriedrich**
Addon-info for new add-on suggestion finder which detects running processes on the OH server.
Refs: openhab/openhab-core#3904

### CODE REVIEW

**Review by ccutrer — APPROVED** [review]

File: bundles/org.openhab.binding.onewire/src/main/resources/OH-INF/addon/addon.xml [hunk 1]
```diff
@@ -6,4 +6,27 @@
...
+		<discovery-method>
+			<service-type>mdns</service-type>
+			<mdns-service-type>_owserver._tcp.local.</mdns-service-type>
+			<match-properties>
+				<match-property>
+					<name>name</name>
+					<regex>OWFS (1-wire) Server</regex>
+				</match-property>
+			</match-properties>
+		</discovery-method>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.onewire/src/main/resources/OH-INF/addon/addon.xml:20]: Can you put that in a separate PR so that it can be merged quickly.
Is there a benefit to match the name while the service seems already very specific ?

Comment (holgerfriedrich) [reply bundles/org.openhab.binding.onewire/src/main/resources/OH-INF/addon/addon.xml]: Hm, I just tried to confirm, but a newly installed instance of owserver on my test system did not announce via mdns at all. I have removed this part for now.

**Review by andrewfg — APPROVED** [review]
I did not review the actual regexes, but from an architecture POV it LGTM.

File: features/openhab-addons-external/pom.xml [hunk 2]
```diff
@@ -36,6 +36,7 @@
                   <header file="src/main/resources/addon-header.xml" filtering="no"/>
                   <fileset dir="${basedirRoot}/bundles">
                     <include name="*/src/main/resources/OH-INF/addon/addon.xml"/>
+                    <include name="*/src/main/resources/OH-INF/addon/addon-*.xml"/>
```
Comment (mherwege) [inline features/openhab-addons-external/pom.xml:39]: Sorry, created a separate PR https://github.com/openhab/openhab-addons/pull/16044 for this. I hadn't seen you included it here.
You could simplify this to one line:
`<include name="*/src/main/resources/OH-INF/addon/addon*.xml"/>`

Comment (holgerfriedrich) [reply features/openhab-addons-external/pom.xml]: rebased

**Comment by mherwege** [comment]
LGTM

**Review by mherwege — APPROVED** [review]
